servant-js/examples/counter.hs

```diff
@@ -43,7 +43,7 @@ type TestApi = "counter" :> Post '[JSON] Counter -- endpoint for increasing the
           :<|> "counter" :> Get '[JSON] Counter -- endpoint to get the current value
 
 type TestApi' = TestApi
-           :<|> Raw -- used for serving static files
+          :<|> Raw IO Application -- used for serving static files
```

Comment:
I'm sorry but I don't have any good answer. I still think we shouldn't ship this because it's a true hack and I'm sincerely unable to explain what `m` people should pick. :(